### PR TITLE
chore: pass schedulerUuid param to minimal dashboard if it exists

### PR DESCRIPTION
**Pull request by joaoviana**
Closes: #7784 

### Description:

Pass `schedulerUuid` to minimal dashboard params on scheduled deliveries if it exists;
We then can use the scheduler context on Minimal dashboards like so:
```tsx
const { search } = useLocation();
    const searchParams = new URLSearchParams(search);
    const schedulerUuid = searchParams.get('schedulerUuid');
    const { data: scheduler, isLoading } = useScheduler(schedulerUuid ?? '');
```

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/backend/src/scheduler/SchedulerTask.ts [hunk 1]
```diff
@@ -119,14 +125,21 @@ export const getNotificationPageData = async (
     let csvUrl;
     let csvUrls;
     let pdfFile;
+
+    const schedulerUuid =
+        'schedulerUuid' in scheduler &&
+        typeof scheduler.schedulerUuid === 'string'
+            ? scheduler.schedulerUuid
+            : undefined;
```
Comment (joaoviana) [inline packages/backend/src/scheduler/SchedulerTask.ts:133]: added all these checks so it conforms to the backwards compatibility mentioned in: https://github.com/lightdash/lightdash/blob/chore/allow-for-minimal-to-have-access-to-scheduler-uuid/packages/backend/src/scheduler/SchedulerTask.ts#L295
But the `schedulerUuid` value is always set on new and recent scheduled deliveries

File: packages/backend/src/scheduler/SchedulerTask.ts [hunk 2]
```diff
@@ -87,9 +88,14 @@ const getChartOrDashboard = async (
...
             dashboardUuid,
         );
+
         return {
             url: `${lightdashConfig.siteUrl}/projects/${dashboard.projectUuid}/dashboards/${dashboardUuid}/view`,
-            minimalUrl: `${lightdashConfig.siteUrl}/minimal/projects/${dashboard.projectUuid}/dashboards/${dashboardUuid}`,
+            minimalUrl: `${lightdashConfig.siteUrl}/minimal/projects/${
+                dashboard.projectUuid
+            }/dashboards/${dashboardUuid}${
+                schedulerUuid ? `?schedulerId=${schedulerUuid}` : ''
```
Comment (IrakliJani) [inline packages/backend/src/scheduler/SchedulerTask.ts:97]: should param also be `schedulerUuid` as well for consistency?

**Review by IrakliJani — APPROVED** [review]
one comment, otherwise looks good!

File: packages/backend/src/scheduler/SchedulerTask.ts (hunk 2 above)
Comment (joaoviana) [reply packages/backend/src/scheduler/SchedulerTask.ts:97]: oops! good catch, renaming!

**Pull request merged**